packages/server-core/src/appconfig.ts

```diff
@@ -418,7 +419,8 @@ const config = {
   testEnabled,
   /** @todo when project versioning is fully implemented, remove 'undefined' check here */
   allowOutOfDateProjects:
-    typeof process.env.ALLOW_OUT_OF_DATE_PROJECTS === 'undefined' || process.env.ALLOW_OUT_OF_DATE_PROJECTS === 'true'
+    typeof process.env.ALLOW_OUT_OF_DATE_PROJECTS === 'undefined' || process.env.ALLOW_OUT_OF_DATE_PROJECTS === 'true',
+  enableFsProjectSync: process.env.ENABLE_FS_PROJECT_SYNC ?? 'true'
```

Comment:
This `?? 'true'` needs to be `false`, because our deployments would not contain this `ENABLE_FS_PROJECT_SYNC` env and thus for deployments, `enableFsProjectSync` will be true which we dont want.

> With it being false by default, we would need everyone having this running locally to update their `.env.local` with `ENABLE_FS_PROJECT_SYNC=true`

Reply:
@HexaField your thoughts?